modfile.go

```diff
@@ -0,0 +1,69 @@
+package gograveyard
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"strings"
+)
+
+type ModFile struct {
+	Direct   []*Module
+	Indirect []*Module
+}
+
+type Module struct {
+	Path    string
+	Version string
+}
+
+func Parse(data []byte) (*ModFile, error) {
+	scanner := bufio.NewScanner(bytes.NewReader(data))
+
+	var require bool
+	var f ModFile
+
+	for scanner.Scan() {
+		switch scanner.Text() {
+		case "require (":
+			require = true
+
+			continue
```

Comment:
```suggestion
			require = true
			continue
```

Reply:
`continue with no blank line before (nlreturn)go-golangci-lint` I get this linter error complaining about the missing space, should I disable it perhaps?

Reply:
oh interesting! yeah I'm cool with disabling that one as it seems a little weird